index.js

```diff
@@ -110,11 +110,11 @@ client.on('message', async msg => {
 		return
 	}
 
-	// Prevent commands below this from being run after hours
+	// Prevent commands below this from being run outside of mentoring hours
 	let now = new Date();
-	if (now.getHours() < 9 || now.getHours() > 17 || now.getDay() != 5) {
+	if (now.getHours() < 9 || now.getHours() > 17 || now.getDay() > 5) {
 		if (msg.content.toLowerCase().startsWith("!") && !mentor) {
-			msg.channel.send(`Sorry ${msg.author}, but we currently only offer online mentoring on Fridays 10AM-6PM. ` +
+			msg.channel.send(`Sorry ${msg.author}, but we currently only offer online mentoring from 10AM-6PM on Monday through Friday. ` +
 				"However Monday through Thursday we offer mentoring in the SSE 10AM-6PM. " +
```

Comment:
Are both online and in-person mentoring going to be offered?

Reply:
In light of RIT's Orange COVID level, we're online only for the remainder of the term.

Reply:
The offending line claiming in person mentoring has been removed. This was remnant of me messing up the merge conflict because I'm bad at Vim.